tests.js

```diff
@@ -0,0 +1,29 @@
+
+'use strict'
+
+
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import Compuguess from '../src/Compuguess'
+
+chai.use(chaiChange)
+
+/*1: Guess The Number
+Write a program that will generate a number between 1 and 100. Your program will ask the user to guess the generated number.When the user types a guess, the program should tell them if they won (the guess matched the computer's number), if their guess was higher than the number the computer generated, OR if the guess was lower than the number the computer generated. Terminate the program if the user types "exit". */
+
+
+describe('compuGuess()', () => {
+  it('should not generate a number above 100', () => {
+    console.log('Guess the generated number');
+    expect()).to.equal('1-100')
+
+  it('should tell them if they won')
+  })
```

Comment:
where is the expect to check if the user has won? This test is incomplete without the expect.